doc/modules/clustering.rst

```diff
@@ -1388,3 +1388,30 @@ Drawbacks
   concepts of clusters, such as density based clusters like those obtained
   through DBSCAN.
 
+.. _contingency_matrix:
+
+Contingency Matrix
+----------------------
+
+Contingency matrix is a format displaying the multivariate frequency distribution of the variables. In a typical contingency matrix, rows are true labels, and columns are predicted labels, which can be different from true labels. Each cell represents the interaction of true and predicted labels.
```

Comment:
Maybe of two discrete variables in the first sentence? Also please break the line at 79 characters. Also I wouldn't talk about a "typical" contigency matrix. We only need to talk about the way it is in scikit-learn.